apps/web/utils/actions.ts

```diff
@@ -106,6 +125,9 @@ export async function deleteAccountAction() {
     captureException(error);
   }
 
+  if (lemonSqueezySubscriptionItemId)
+    await cancelSubScriptionForUser(lemonSqueezySubscriptionItemId);
+
   await prisma.user.delete({ where: { email: session.user.email } });
```

Comment:
I think we should delete the user after the subscription tenure is ended.

Reply:
No, don't delete the user. "leaving a user's data in Stripe after they delete their account is generally allowed under GDPR, provided that you have a legitimate reason for retaining this data, such as for financial records, compliance with tax law, or other regulatory requirements."

Reply:
For now i have commented that part